app/boards/shields/qaz/qaz.conf

```diff
@@ -0,0 +1,26 @@
+CONFIG_ZMK_RGB_UNDERGLOW=n
+# Use the STRIP config specific to the LEDs you're using
+CONFIG_WS2812_STRIP=n
+# Turn on logging, and set ZMK logging to debug output
+CONFIG_LOG=y
+CONFIG_ZMK_LOG_LEVEL_DBG=y
+
+# Turn on USB CDC ACM device
+CONFIG_USB=y
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_CDC_ACM=y
+CONFIG_USB_CDC_ACM_RINGBUF_SIZE=1024
+CONFIG_USB_CDC_ACM_DEVICE_NAME="CDC_ACM"
+CONFIG_USB_CDC_ACM_DEVICE_COUNT=1
+
+# Enable serial console
+CONFIG_SERIAL=y
+CONFIG_CONSOLE=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_LINE_CTRL=y
+
+# Enable USB UART, and set the console device
+CONFIG_UART_CONSOLE=y
+CONFIG_USB_UART_CONSOLE=y
+CONFIG_UART_CONSOLE_ON_DEV_NAME="CDC_ACM_0"
+CONFIG_USB_UART_DTR_WAIT=n
```

Comment:
We have *not* been turning on logging by default in these files, since it is a theoretically security concern, since it logs keystroke information. These should be removed for the version of this for in-tree use.

Reply:
oops forgot I had that in there